protocol/messenger_communities.go

```diff
@@ -18,6 +19,9 @@ import (
 	"github.com/status-im/status-go/protocol/transport"
 )
 
+// 7 days interval
+var messageArchiveInterval = 7 * 24 * time.Hour
```

Comment:
For testing purposes, it might make sense to change this for now or at least make it configurable.